### PR TITLE
Make it build with ghc-9.6

### PR DESCRIPTION
**Pull request by erikd**

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -13,3 +13,22 @@ source-repository-package
 package inspector
   -- This package uses bang patterns without enabling them
   ghc-options: -XBangPatterns
+
+-- This is not on Hackage.
+source-repository-package
+  type: git
+  location: https://github.com/primetype/inspector
+  tag: 962c6b5b52b760eeeb847d318800392be9aab5fa
```
Comment (newhoggy) [inline cabal.project:35]: Shouldn't this be forked into `input-output-hk` organisation and published to CHaP?

Comment (erikd) [reply cabal.project:35]: Its only used in the tests (for this package), so we do not need it in CHaP.

I have however fixed it to point at the `input-output-hk` version.

Comment (newhoggy) [reply cabal.project:35]: Please add comment to that effect.

**Comment by erikd** [comment]
Should not merge this before the `foundation` issue is resolved. https://github.com/haskell-foundation/foundation/pull/572

File: cabal.project (hunk 1 above)
Comment (angerman) [reply cabal.project:35]: Do we have a policy that explicitly says we should not push test dependencies to CHaP?

**Review by newhoggy — APPROVED** [review]

**Pull request merged**